### PR TITLE
Add supported Luci builders for packages repo

### PR DESCRIPTION
**Pull request by keyonghan**
Existing Luci builder configurations are stored in cocoon project. This is not efficient because we need to re-deploy the flutter-dashboard after changing the builder configuration. Additionally, with different channels/branches, a single place becomes challenging to support various requests.

Moving build configuration files to `packages` repo by itself will make things easier. Repos/branches will maintain their own copy of builder config file. From cocoon side, it can directly obtain the build config file for different repos/branches without redeploying the app engine.

JSON file will be validated via luci presubmit test, to be defined in fuchsia_ctl.py recipe.

Related issues: flutter/flutter#62429, flutter/flutter#53371

### CODE REVIEW

File: dev/packages_try_builders.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (godofredoc) [inline dev/packages_try_builders.json:1]: can we use the same paths in all the repos? e.g in the engine repo we are using ci/dev/... additionally let's use just try_builders.json to make the recipes code more generic.

Comment (keyonghan) [reply dev/packages_try_builders.json:1]: I prefer to use the same path as well. 
For engine, as suggested by Yuqian, I saved it to ci/dev.
For sdk, we already have dev. I am not sure if it is okay/good to create a ci from root.
For packages, no dev or ci at all. I guess we can create either way.
For cocoon, it is now in dev, but up to change.

So, it is engine and sdk. what do you think?

Comment (godofredoc) [reply dev/packages_try_builders.json:1]: \cc @goderbauer is it ok to add a new top level folder called ci in this repo? this is to use the same path for builder configurations across repos.

Comment (goderbauer) [reply dev/packages_try_builders.json]: Creating a dev folder in this repo sounds good.

**Pull request merged**